curriculum/en/6-building-your-frontend/1-intro-to-rainbowkit.md

```diff
@@ -15,9 +15,24 @@ You can fully customize your RainbowKit theme and include only the necessary fea
 
 ## Loading Environment Variables
 
-At the root of your project, create a new file called `.env.local`. In the web3rsvp-frontend-starter, there is a file called **.env.example** that shows an example of how to set up your .env.local file. This file is where we will keep secrets like our API keys so they aren't exposed on the frontend.
+At the root of your project, create a new file called `.env.local`. You can use the `touch .env.local` command in your terminal to create this file. In the web3rsvp-frontend-starter, there is a file called **.env.example** that shows an example of how to set up your .env.local file. This file is where we will keep secrets like our API keys so they aren't exposed on the frontend. 
 
-While you're in this file, you can also replace `<Your Infura project id>` with your Infura project id. You can find that by going to your Infura dashboard and selecting your project settings.
+> You're going to need the Web3storage API token to complete the `.env.local` file. Skip ahead to the Web3storage section [here](https://www.30daysofweb3.xyz/en/curriculum/6-building-your-frontend/3-intro-to-web3storage) to create an account. Once you generate an API token, come back to this section to finish setup. 
```

Comment:
since the web3storage token isn't required for the rainbow integration to work, maybe we can reword this to something along the lines of "don't worry about web3storage token we'll be setting up webstorage later in section 6"

also kindly add yourself to the list of editors at the end of this page and 3-writing-your-smart-contract/1-dev-setup.md :)